README.md

```diff
@@ -11,16 +11,77 @@ The underlying tools are under one or more Free and Open Source Software license
 
 # Styleguide
 
-1. First line must be `#!/usr/bin/env cwl-runner`
-1. Second line is `cwlVersion: v1.0`
-1. Third line is `class: CommandLineTool`
-1. `*.cwl` files must be marked executable (`chmod a+x *.cwl`)
-1. All tool descriptions must have a software container. Use a container from biocontainers.pro if available
-1. If you use schema.org annotations, specify the schema using the RDF version: `$schemas: [ http://schema.org/version/latest/schema.rdf ]`
-1. Must be free from warning when `cwltool --validate` is run
-1. Tool descriptions should be motivated by a real world use of this tool in a workflow.
-   The description should focus on a single way of using the tool.
-   Signs that a tool description is including too much: lots of javascript; complicated data structures; every single flag is listed.
+## Naming Tools 📛
+
+Tools should follow the convention of being prefixed by the parent tool name and Camelcase like so i.e.
```

Comment:
usually I prefer lowercase for faster typing. Thoughts from other @common-workflow-library/bio-team people?

Reply:
We used camelcase so we can add some more descriptive naming from the description in case of a long form parameter.

Reply:
I usually use lowercase letters and underscore if I want to separate parts of the name, but it mostly defined by a habit and non-case-sensitive hard-drive.

Reply:
So maybe this is something we can keep as part of our internal style guide and have it be something open for people’s coding habits?

Reply:
Yeah, I think this may be premature optimization right now

Reply:
@mr-c Should we merge this PR or there is something left to fix?

Reply:
@mr-c I believe this is good.